### ISSUE TITLE
NVDA not able to read the checkbox/radio button status in case of static non interactive(print form) pdf

### ISSUE THREAD

**Issue by cprabhak1** [nvaccess/nvda#13285]
### Steps to reproduce:
Open a static non interactive pdf containing checkbox. Try to read the state of checkbox using NVDA.
### Actual behavior:
NVDA does not read check/not checked
### Expected behavior:
NVDA  should speak checkbox checked(if checked) or vise versa
### System configuration
windows 10, pdf 1.7
#### NVDA version:
NVDA 2020.4
#### Windows version:
windows 10
#### Name and version of other software in use when reproducing the issue:
Adobe Acrobat dc pro
### Other questions
####Is the issue reproducible for interactive static pdf as well? 
No

###Is the issue reproducible on Jaws?
NO 
[output_non.pdf](https://github.com/nvaccess/nvda/files/7958484/output_non.pdf)

#### Does the issue still occur after restarting your computer?
yes

**Comment by michaelDCurran** [comment]
Confirmed this is a bug in NVDA.
We are incorrectly ignoring the accRole and accState for checkboxes. 
This is a small fix which we will get to soon.

**Comment by michaelDCurran** [comment]
Technical:
The checkbox has an Adobe STDName of 'Form'. We incorrectly map this to ROLE_FORM, where as the accRole is actually much more accurate as ROLE_SYSTEM_CHECKBUTTON.
We should remove the 'Form' mapping as it is hiding this and other possible controls.
